src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -36,18 +35,16 @@ class DebugHandlersListener implements EventSubscriberInterface
     private $levels;
     private $throwAt;
     private $scream;
-    private $fileLinkFormat;
     private $scope;
     private $firstCall = true;
     private $hasTerminatedWithException;
 
     /**
-     * @param callable|null                 $exceptionHandler A handler that must support \Throwable instances that will be called on Exception
-     * @param array|int                     $levels           An array map of E_* to LogLevel::* or an integer bit field of E_* constants
-     * @param int|null                      $throwAt          Thrown errors in a bit field of E_* constants, or null to keep the current value
-     * @param bool                          $scream           Enables/disables screaming mode, where even silenced errors are logged
-     * @param string|FileLinkFormatter|null $fileLinkFormat   The format for links to source files
-     * @param bool                          $scope            Enables/disables scoping mode
+     * @param callable|null $exceptionHandler A handler that must support \Throwable instances that will be called on Exception
+     * @param array|int     $levels           An array map of E_* to LogLevel::* or an integer bit field of E_* constants
+     * @param int|null      $throwAt          Thrown errors in a bit field of E_* constants, or null to keep the current value
+     * @param bool          $scream           Enables/disables screaming mode, where even silenced errors are logged
+     * @param bool          $scope            Enables/disables scoping mode
      */
     public function __construct(callable $exceptionHandler = null, LoggerInterface $logger = null, $levels = \E_ALL, ?int $throwAt = \E_ALL, bool $scream = true, $fileLinkFormat = null, bool $scope = true, LoggerInterface $deprecationLogger = null)
```

Comment:
we should allow using the new signature now
this means we should remove the argument, and use func_get_arg + type checks to decide if the old signature is used or not and restore the appropriate behavior

the alternative is to make this class `@internal` since 5.3 and do what we want in v6...